common/placeholders/src/main/java/io/knotx/server/common/placeholders/JsonPlaceholderSubstitutor.java

```diff
@@ -0,0 +1,17 @@
+package io.knotx.server.common.placeholders;
+
+import static io.knotx.commons.json.JsonObjectUtil.getString;
+
+import org.apache.commons.lang3.StringUtils;
+
+import io.vertx.core.json.JsonObject;
+
+public class JsonPlaceholderSubstitutor implements PlaceholderSubstitutor<JsonObject> {
+
+  static final String PREFIX_PAYLOAD = "payload.";
```

Comment:
I wonder if this is proper prefix...
Knot.x Server does not know anything about Knot.x Fragments, and `payload` is part of a fragment.
I'm also not 100% satisfied with prefix like `json.` - maybe you have any better idea?

Reply:
not really. I assumed that payload is generic enough :)

Reply:
Ok, let's leave it for now.